.gitignore

```diff
@@ -36,4 +36,5 @@ blackbox.zip
 blackbox
 rllab/config_personal.py
 *.swp
-sandbox
+sandbox/
```

Comment:
Once we finish porting sandbox.rocky.tf to rllab.tf, this will not be necessary right?

Reply:
as long as garage uses conda instead of pip it will probably be necessary. i keep my private research code in a sandbox checked out into a sandbox (and so do most people).

Reply:
sgtm

Reply:
you forgot to approve :P

Reply:
whoops, my bad